src/js/components/Tag/Tag.js

```diff
@@ -7,12 +7,32 @@ import { Text } from '../Text';
 
 import { StyledRemoveButton, StyledTagButton } from './StyledTag';
 import { useThemeValue } from '../../utils/useThemeValue';
+import { MessageContext } from '../../contexts/MessageContext';
 
 const Tag = forwardRef(
-  ({ name, value, size = 'medium', onRemove, onClick, ...rest }, ref) => {
+  (
+    {
+      name,
+      value,
+      size = 'medium',
+      onRemove,
+      onClick,
+      messages,
+      removeProps = { 'aria-label': '' },
```

Comment:
```suggestion
```
Do we even need this anymore since it's handled my `messages`?

Reply:
aw yes good catch